Engine/source/T3D/physics/physicsShape.cpp

```diff
@@ -271,6 +271,8 @@ bool PhysicsShapeData::preload( bool server, String &errorBuffer )
 {
    if ( !Parent::preload( server, errorBuffer ) )
       return false;
+   
+   if( server ) return true;
```

Comment:
This needs to be removed, it stops the TSShape and PhysicsCollisionRef from been created server side.